index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <main>
+        <nav>
+          <a href="#">Ezema Anthony</a>
+          <a><img src="img/Unionunioin.png" alt=""></a>
+        </nav>
+        <h1>I’m Anthony Glad to see you!</h1>
```

Comment:
- [ OPTIONAL ] I recommed  adding some font weight to make it simialr to the Figma template🙏🏻🚀🔥